msf_updates/src/pose_msf/pose_sensormanager.h

```diff
@@ -170,16 +170,16 @@ class PoseSensorManager : public msf_core::MSF_SensorManagerROS<
     P.setZero();  // Error state covariance; if zero, a default initialization in msf_core is used
 
     p_vc = pose_handler_->GetPositionMeasurement();
-    q_cv = pose_handler_->GetAttitudeMeasurement();
+    q_vc = pose_handler_->GetAttitudeMeasurement();
```

Comment:
Here it says that measurement_world_sensor is false for ptam:
http://wiki.ros.org/ethzasl_sensor_fusion/Tutorials/sfly_framework